common/views/pages/_app.js

```diff
@@ -399,6 +399,28 @@ export default class WecoApp extends App {
                 <CatalogueSearchProvider>
                   <OutboundLinkTracker>
                     <Fragment>
+                      <TogglesContext.Consumer>
+                        {({ helveticaRegular }) =>
+                          helveticaRegular && (
+                            <style
+                              type="text/css"
+                              dangerouslySetInnerHTML={{
+                                __html: `
+                                @font-face {
+                                  font-family: 'Helvetica Neue Light Web';
+                                  src: local('Helvetica Neue Light'),
```

Comment:
Is this local source still meant to be Helvetica Neue Light?

Reply:
Nope. Good spot, thanks. Will update.